src/main/java/com/synopsys/integration/jira/common/model/request/JiraRequestFactory.java

```diff
@@ -32,8 +32,8 @@
 import com.synopsys.integration.rest.HttpUrl;
 
 public class JiraRequestFactory {
-    public static final String LIMIT_PARAMETER = "limit";
-    public static final String OFFSET_PARAMETER = "offset";
+    public static final String LIMIT_PARAMETER = "maxResults";
```

Comment:
Is this the equivalent of "limit" or of "total"?

Reply:
Yes, these are the values they use in their pagination. We had the wrong values originally https://developer.atlassian.com/cloud/jira/platform/rest/v2/intro/#expansion

Reply:
Oh sorry, to answer your question, it's the equivalent of total I believe.

Reply:
Do these apply to both Cloud and Server?

Reply:
Yes, the expansion section at the beginning here goes over the server expectations https://docs.atlassian.com/software/jira/docs/api/REST/7.6.1/